app/views/jobs/dashboard.html.erb

```diff
@@ -0,0 +1,40 @@
+<h1 class="title">Dashboard</h1>
+
+<div class="test">
+  <table class="table">
+    <tr><th>Level of Interest</th><th>Number of Jobs</th></tr>
+    <%= @jobs_by_interest.each do |k, v| %>
```

Comment:
i encourage you to choose better names for `k` and `v`. these are not very descriptive